configuration/spring-config-server/src/main/resources/config/configdemo-dev.yaml

```
@@ -2,5 +2,19 @@
+  rabbitmq:
+    host: localhost
+    port: 5672
+    username: guest
+    password: guest
+
+  # Here define actuator
+management:
+  endpoints:
+    web:
+      exposure:
+        include : "*"
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)